plugins/inputs/t128_graphql/entry_point_test.go

```diff
@@ -0,0 +1,39 @@
+package t128_graphql_test
+
+import (
+	"testing"
+
+	plugin "github.com/influxdata/telegraf/plugins/inputs/t128_graphql"
+	"github.com/stretchr/testify/require"
+)
+
+//TODO: more unit tests - MON-314
+var JSONPathFormationTestCases = []struct {
+	Name                 string
+	EntryPoint           string
+	ExpectedResponsePath string
+	ExpectedQueryPath    string
+	ExpectedPredicates   map[string]string
+}{
+	{
+		Name:                 "build arp state json path",
+		EntryPoint:           "allRouters[name:ComboEast]/nodes/nodes[name:east-combo]/nodes/arp/nodes",
+		ExpectedResponsePath: "/data/allRouters/nodes/0/nodes/nodes/0/arp/nodes",
```

Comment:
Nbd but you could make this a `ParsedEntryPoint` and then you do a `require.Equal` between the two structs.